packages/jsts/tests/analysis/analyzer.test.ts

```diff
@@ -123,7 +123,7 @@ describe('analyzeJSTS', () => {
     );
   });
 
-  it('should analyze Vue.js code with type checks', async () => {
+  it('Vue.js analysis does not support type checking', async () => {
```

Comment:
should it read more like a sentence?
```suggestion
  it('should not analyze Vue.js with type checks', async () => {
```